backend/core/src/applications/services/applications.service.ts

```diff
@@ -251,6 +255,25 @@ export class ApplicationsService {
     return await this.repository.softRemove({ id: applicationId })
   }
 
+  async sendExport(queryParams: ApplicationsCsvListQueryParams): Promise<StatusDto> {
```

Comment:
I think we may want `authorizeUserAction` here?

Reply:
Hey @emilyjablonski great question. In `rawListWithFlagged()` (the first function `sendExport()` calls) we make a call to `authorizeCSVExport()` which verifies the user has permissions to get the export of applications. 

Hope that helps!